packages/manager/apps/pci-quota/src/constants.ts

```diff
@@ -0,0 +1,35 @@
+export const PRODUCTS = {
+  FLOATING_IP: 'Floating IPs',
+  GATEWAYS: 'Gateways',
+  LB_OCTAVIA: 'LB Octavia',
+};
+
+export const QUOTA_THRESHOLD = 80;
+
+export const iamLink = 'https://www.ovh.com/manager/#/dedicated/iam/policy';
+export const supportLink = 'https://us.ovhcloud.com/manager/#/dedicated/ticket';
```

Comment:
Subsidiary should be handled

If these are constant they should be in upper snake case